ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/teleoperation/RDXHardwareControlStateManager.java

```diff
@@ -60,22 +69,24 @@ public void renderImGuiWidgets(DRCRobotModel robotModel, HumanoidReferenceFrames
          boolean changed = ImGui.checkbox(labels.get("SOFT-E-STOP"), estop);
          ImGui.sameLine();
          ImGui.setNextItemWidth(ImGui.getColumnWidth());
-         changed |= ImGuiTools.sliderDouble(labels.getHidden("Master Gain"), masterGain, 0.0, 1.0, "Master Gain: %.2f");
+         changed |= ImGuiTools.sliderDouble(labels.getHidden("Master Gain"), desiredMasterGain, 0.0, 1.0, "Master Gain: %.2f");
          boolean servoRobot = ImGui.button(labels.get("Servo Robot"));
          changed |= servoRobot;
          ImGui.sameLine();
-         boolean unservoSlowly = ImGui.button(labels.get("Unservo Slowly"));
-         changed |= unservoSlowly;
+         boolean unservoQuickly = ImGui.button(labels.get("Unservo Quickly"));
+         changed |= unservoQuickly;
```

Comment:
Can you change this button back to unservoSlowly? The eStop checkbox already serves as unservo quickly